src/Entity/OgRole.php

```diff
@@ -243,6 +252,13 @@ public function delete() {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function isDefaultRole() {
+    return in_array($this->getName(), [self::ANONYMOUS, self::AUTHENTICATED]);
```

Comment:
Very good point. `isDefaultRole()` is not the correct name for this method.

Maybe a better name would be `isRequiredRole()`, then it explains more clearly what this returns, and why you would need to call this method.